sdk/docs/core/resources/mqtt_connection.puml

```diff
@@ -40,11 +40,11 @@ state ConnectionManagement {
         Disconnecting : <b>AZ_EVENT_MQTT5_CONNECTION_CLOSE_REQ</b>/ ignore
 
         [*] -> Connecting
-        Connecting --> Connected : <b>MQTT_CONNECT_RSP_OK</b>/\n send_inbound(MQTT_CONNECT_RSP_OK)
+        Connecting --> Connected : <b>MQTT_CONNECT_RSP_OK</b>/\n send_inbound(MQTT_CONNECTION_OPEN_RSP)
```

Comment:
Do these changed need to be made in the samples too?

Reply:
Good point! I will run those and make the edits